db/migrate/20151435234623_add_container_command.rb

```diff
@@ -0,0 +1,5 @@
+class AddContainerCommand < ActiveRecord::Migration
+  def change
+    add_column :container_definitions, :command, :string
```

Comment:
@blomquisg ah good catch! Is there something more text/blob-like? I fear that limit is 4096 or even higher.